911samuel/instagram-clone
# ft-home-screen

#### What does this PR do?

This PR implements the home screen of the Instagram clone, including the feed of posts, header, and bottom navigation.

#### Description of Task to be completed?

- Implemented the HomeScreen component.
- Integrated the Header and BottomTab components.
- Fetched post data and displayed it in the feed.
- Implemented navigation between screens.

#### How should this be manually tested?

1. Ensure the app is running on an iOS/Android device or simulator/emulator.
2. Navigate to the home screen.
3. Verify that the feed of posts is displayed correctly.
4. Check that the header and bottom navigation are functional.
5. Test navigation to other screens.

#### Any background context you want to provide?

This PR is part of the ongoing development of the Instagram clone project. It focuses on building the core functionality of the home screen.

#### Screenshots (if appropriate)

![home](https://github.com/911samuel/instagram-clone/assets/114469939/db3f044e-3ec4-410b-8ba8-e39166e64458)

#### Questions:

Any questions or feedback on the implementation?
